anypoint-signin-aware.js

```diff
@@ -680,6 +706,15 @@ export class AnypointSigninAware extends LitElement {
     AnypointAuth.clientId = newId;
   }
 
+  /**
+   * Sets AnypointAuth with an array of scopes, e.g. ['full','profile','email']
+   * @param {String} newScopes space separated scopes, e.g. 'full profile email'
+   */
+  _scopesChanged(newScopes) {
```

Comment:
Note my comment for `scope` property. If you decide to support array values then this should be working with array. Array values can be passed as a property (not by the attribute setting), in a templating system for example (React, LitElement, Polymer).

Reply:
Decided to support string only for now